src/Microsoft.CSharp/src/ILLinkTrim.xml

```diff
@@ -0,0 +1,8 @@
+<linker>
+  <assembly fullname="Microsoft.CSharp">
+    <!-- Required by the Expression Evaluator -->
+    <type fullname="Microsoft.CSharp.RuntimeBinder.DynamicMetaObjectProviderDebugView"/>
+    <!-- Required by the Expression Evaluator -->
+    <type fullname="Microsoft.CSharp.RuntimeBinder.DynamicMetaObjectProviderDebugView/DynamicDebugViewEmptyException"/>
```

Comment:
Just to make sure, you've confirmed that these types exist in the release binary?

Reply:
Yes, the unit tests rely on these types.